src/isomorphic/classic/element/ReactCurrentOwner.js

```diff
@@ -11,20 +11,6 @@
 
 'use strict';
 
-/**
- * Keeps track of the current owner.
- *
- * The current owner is the component who should own any components that are
- * currently being constructed.
- */
-var ReactCurrentOwner = {
-
-  /**
-   * @internal
-   * @type {ReactComponent}
-   */
-  current: null,
-
-};
+var ReactCurrentOwner = require('react-current-owner');
```

Comment:
Remotefriendlycoughcough

Would abusing the RN packager be analogous to haste tooling? I dug around a bit today and noticed this current impl is a bit lacking.